react/locales/pt-BR.json

```diff
@@ -32,13 +40,18 @@
   "editor.footer.title": "Rodapé",
   "editor.footer.description": "Um rodapé que aparece no inferior da sua loja",
   "editor.footer.logoUrl.title": "URL da logo",
-  "editor.footer.numberOfSocialNetworks.title": "Número de links de redes sociais",
+  "editor.footer.numberOfSocialNetworks.title":
+    "Número de links de redes sociais",
   "editor.footer.numberOfSectionLinks.title": "Número de links de seções",
-  "editor.footer.numberOfMoreInformationLinks.title": "Número de links para mais informações",
+  "editor.footer.numberOfMoreInformationLinks.title":
+    "Número de links para mais informações",
   "editor.footer.numberOfBadges.title": "Número de medalhas a serem exibidas",
-  "editor.footer.numberOfPaymentForms.title": "Número de formas de pagamento disponíveis",
-  "editor.footer.showPaymentFormsInColor.title": "Mostrar os ícones das formas de pagamento coloridos",
-  "editor.footer.showSocialNetworksInColor.title": "Mostrar os ícones das redes sociais coloridos",
+  "editor.footer.numberOfPaymentForms.title":
```

Comment:
Cammel case or `-` case, the decision to the pattern was **Cammel Case**

Reply:
I fixed the Price ones.